src/PostgREST/Response/OpenAPI.hs

```diff
@@ -175,8 +175,25 @@ makePreferParam ts =
       "resolution" -> ["resolution=ignore-duplicates", "resolution=merge-duplicates"]
       _            -> []
 
-makeProcParam :: ProcDescription -> [Referenced Param]
-makeProcParam pd =
+makeProcGetParam :: ProcParam -> Referenced Param
+makeProcGetParam (ProcParam n t r v) =
+  Inline $ (mempty :: Param)
+    & name .~ n
+    & required ?~ r
+    & schema .~ ParamOther ((mempty :: ParamOtherSchema)
+      & in_ .~ ParamQuery
+      & type_ .~ toSwaggerType t
+      & items .~ if not v then Nothing else Just $ SwaggerItemsPrimitive
+          (Just CollectionMulti)
+          ((mempty :: ParamSchema x)
+            & type_ .~ toSwaggerType (T.dropEnd 2 t)
+            & format ?~ t))
```

Comment:
This allows for `variadic` parameters to be repeated in the Swagger UI. 

![image](https://user-images.githubusercontent.com/18169764/217296083-9c41f7f2-c69d-40eb-a4fa-b3458a85abbd.png)

I tried to show the format for special types (for instance a SQL `ENUM`) as a visual help, but while adding it outside `items` shows the format, the parameters can no longer be repeated. Adding it inside `items` (like it is right now) doesn't change the behavior, but the visual help is not shown. I think I'll remove it in the refactor since it doesn't seem to help.